crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -666,15 +669,26 @@ pub struct UnableToDecryptInfo {
     pub session_id: Option<String>,
 
     /// Reason code for the decryption failure
-    #[serde(default = "unknown_utd_reason")]
+    #[serde(default = "unknown_utd_reason", deserialize_with = "ok_reason_or_default")]
     pub reason: UnableToDecryptReason,
 }
 
 fn unknown_utd_reason() -> UnableToDecryptReason {
     UnableToDecryptReason::Unknown
 }
 
-/// Reason code for a decryption failure
```

Comment:
I think removing the documentation on `UnableToDecryptReason` is probably an accident?